urdf2webots/parserURDF.py

```diff
@@ -949,6 +990,42 @@ def parseGazeboElement(element, parentLink, linkList):
                 if hasElement(noiseElement, 'stddev'):
                     camera.noise = float(noiseElement.getElementsByTagName('stddev')[0].firstChild.nodeValue)
             Camera.list.append(camera)
+        elif sensorElement.getAttribute('type') == 'depth':
+            rangefinder = RangeFinder()
+            rangefinder.parentLink = parentLink
+            rangefinder.name = sensorElement.getAttribute('name')
+            if hasElement(sensorElement, 'camera'):
+                cameraElement = sensorElement.getElementsByTagName('camera')[0]
+                if hasElement(cameraElement, 'horizontal_fov'):
+                    rangefinder.fov = float(cameraElement.getElementsByTagName('horizontal_fov')[0].firstChild.nodeValue)
+                if hasElement(cameraElement, 'image'):
+                    imageElement = cameraElement.getElementsByTagName('image')[0]
+                    if hasElement(imageElement, 'width'):
+                        rangefinder.width = int(imageElement.getElementsByTagName('width')[0].firstChild.nodeValue)
+                    if hasElement(imageElement, 'height'):
+                        rangefinder.height = int(imageElement.getElementsByTagName('height')[0].firstChild.nodeValue)
+                if hasElement(cameraElement, 'clip'):
+                    clipElement = cameraElement.getElementsByTagName('clip')[0]
+                    if hasElement(clipElement, 'near'):
+                        rangefinder.near = float(clipElement.getElementsByTagName('near')[0].firstChild.nodeValue)
+            if hasElement(sensorElement, 'noise'):
+                noiseElement = sensorElement.getElementsByTagName('noise')[0]
+                if hasElement(noiseElement, 'stddev'):
+                    rangefinder.noise = float(noiseElement.getElementsByTagName('stddev')[0].firstChild.nodeValue)
```

Comment:
```suggestion
                    rangefinder.noise = float(noiseElement.getElementsByTagName('stddev')[0].firstChild.nodeValue)
                    if rangefinder.maxRange:
                            rangefinder.noise /= rangefinder.maxRange
```

With this change, the `noise` element should be parsed after the `range` one.

Reply:
This part is still missing.

Reply:
Yes, didn't had time to review these changes on my side.